src/fluent-api/playwright-fluent.ts

```diff
@@ -1824,20 +1824,39 @@ export class PlaywrightFluent implements PromiseLike<void> {
     return result;
   }
 
-  public runStory<T>(story: Story | StoryWithProps<T>, param?: T): PlaywrightFluent {
+  private registerStory<T>(story: Story | StoryWithProps<T>, param?: T): PlaywrightFluent {
     if (typeof story !== 'function') {
       throw new Error('Story should be a function');
     }
 
     if (param === undefined) {
       this.actions.push(async (): Promise<void> => await (story as Story)(this));
-      return this;
+    } else {

```

Comment:
Hi @osolomin90, maybe you did not notice that the code in this repo has no `else {}` statements.
Basic rule of thumb in this repo : an `if` statement has never an `else` statement.

Reply:
Indeed. I will not use else any more.